packages/gatsby-theme-docs/src/components/release-notes-filter-dates.js

```diff
@@ -27,11 +31,9 @@ const ReleaseNotesFilterDates = () => {
       <SpacingsStack scale="xs">
         <DateLabel htmlFor="from-filter-date">From</DateLabel>
         <div>
-          <input
-            type="date"
+          <DateInput
```

Comment:
General question: since there are from/to date fields, is there any reason for not using the `DateRangeInput`?

https://uikit.commercetools.com/?path=/story/components-inputs--daterangeinput

Reply:
See the [design](https://commercetools.invisionapp.com/d/main?origin=v7#/console/19438556/408192223/preview?scrollOffset=215)

Reply:
Thanks but that does not answer my question 😅

Reply:
`DateRangeInput` does not meet the needs of the design implementation.

Reply:
Still not answering my question, which is not about the implementation 🙂

Let me rephrase that.

Given that that the filter is a "range" filter, what is the reason for not using a `DateRangeInput`? Is it just because of some design constraints or is the reason about usability?

Reply:
@emmenko  please clarify this with design - they know the kit well enough and chose two separate boxes. 

I personally think two separate inputs are better because
 * if you squeeze the UI kit range component into the width of the index nav column the second date is partly not visible. 
 * typical use cases just need the start date of the list - forcing the user to select two although they don't need to is worsening the UX.